lib/game_client/ogs/ogs_game.dart

```diff
@@ -47,13 +59,79 @@ class OGSGame extends Game {
   Future<void> manualCounting() => Future.value();
 
   @override
-  Future<void> move(wq.Move move) => Future.value();
+  Future<void> move(wq.Move move) {
+    final sgfMove = move.p.toSgf();
+    return _sendMove(sgfMove, move.col);
+  }
 
   @override
-  Stream<wq.Move?> moves() => Stream.empty();
+  Future<void> pass() {
+    return _sendMove('', myColor);
+  }
+
+  Future<void> _sendMove(String sgfMove, wq.Color color) {
+    // Note: OGS Websocket supports 2-way communication if one appends a requestId
+    // we could potentially wait for confirmation of the move being accepted
+    _webSocketManager.send('game/move', {
+      'game_id': int.parse(id),
+      'move': sgfMove,
+    });
+
+    _logger.fine('Sent move "$sgfMove" for game $id');
+
+    return Future.value();
+  }
 
   @override
-  Future<void> pass() => Future.value();
+  Stream<wq.Move?> moves() => _moveController.stream;
+
+  void _setupGame() {
+    // Connect to the game via WebSocket
+    _webSocketManager.joinGame(id);
+
+    // Listen for move events
+    _messageSubscription = _webSocketManager.messages.listen(_handleMessage);
+  }
+
+  void _handleMessage(Map<String, dynamic> message) {
+    final event = message['event'] as String;
+
+    // Check if this is a move event for our game
+    if (event == 'game/$id/move') {
+      final data = message['data'] as Map<String, dynamic>;
+      final gameId = data['game_id'] as int;
+      final moveNumber = data['move_number'] as int;
+      final moveData = data['move'] as dynamic;
+
+      _logger.fine(
+          'Received move: game_id=$gameId, move_number=$moveNumber, move=$moveData');
+
+      // Determine the color based on move number (odd = black, even = white)
```

Comment:
isn't this returned by the server somehow? the parity could change if one player passes and the other one doesn't.